File: practice/5.py
# FIND PRIME NUMBER 


n = int(input("Enter number"))

x = False

for i in range(2,n):
    if n%i == 0:
        x = True

if(x):
    print(f"{n} is not a prime number")
else:
    print(f"{n} is a prime number")

File: practice/5.c
// find prime number 

#include <stdio.h>

int main(){
    int n;
    int isPrime = 1;
    printf("Enter number to check prime or not : ");
    scanf("%d", &n);

    for (int i=2; i<n; i++){
        if(n % i == 0){
           isPrime = 0;
        }
    }
    if(isPrime == 0){
        printf("Not Prime number");
    } else {
        printf("Prime number");
    }

    return 0;
}